### ISSUE TITLE
API reference documentation

### ISSUE THREAD

**Issue by sodrooome** [bocadilloproject/bocadillo#316]
**Is your feature request related to a problem? Please describe.**
when I read the API reference documentation in the `application.py` section, there seems to be a sentence that has not been linked to the `websocket` section/guide.

**Describe the solution you'd like**
it might be better if the documentation is corrected.

**Describe alternatives you've considered**
NA

**Implementation ideas**
Unfortunately, I don't know whether it's a typo or not 😄

**Additional context**
NA

**Comment by florimondmanca** [comment]
Hi! Thanks for reporting this. I noticed there was a strange-looking "::bocadillo.websockets#WebSocket" in .add_websocket_route(), is this what you were referring to? I pushed a fix in #317. :-)

There are probably other examples of this which are caused by a change in pydocmd (the library used to convert docstrings to Markdown files).

If you see more, feel free to try and fix them based on the example in #317 😄

The format for links to the API documentation is "/api/<module_name>.md". For the guide, it is "/guide/<name>.md" (<name> being one of the Markdown files in docs/guide).
